.github/workflows/ci.yml

```diff
@@ -271,7 +271,7 @@ jobs:
 
       # Run clippy
       - name: "clippy --all"
-        run: cargo clippy --all --tests --all-features
+        run: cargo clippy --all --tests --all-features -- -D warnings
```

Comment:
This is not needed. We have already denied warnings (via RUSTFLAGS).

Reply:
Already fixed it :smile:.

Reply:
I don't think you've pushed that change.

Reply:
```suggestion
        run: cargo clippy --all --tests --all-features
```

Reply:
Sorry. I forgot to push it.